### PR TITLE
Added tests for expectNoAttributeValue

### PR DESCRIPTION
**Pull request by Ad1B3nAr0ya**

### CODE REVIEW

File: src/test/specs/ExpectNoAttributeValueSpec.ts [hunk 1]
```diff
@@ -18,4 +18,14 @@ describeCommon('expectNoAttributeValue', () => {
   it('Exact match error thrown', () => {
     expect(() => BrowserUtils.expectNoAttributeValue(EMPTY_DIV, 'align', 'center')).to.throw(Error);
   });
+
+  it('Case Element not found.', () => {
```
Comment (FelixZilber) [inline src/test/specs/ExpectNoAttributeValueSpec.ts]: suggestion for name: "not existing element"

File: src/test/specs/ExpectNoAttributeValueSpec.ts [hunk 2]
```diff
@@ -18,4 +18,14 @@ describeCommon('expectNoAttributeValue', () => {
...
+    BrowserUtils.navigateToUrl(sampleAppUrl);
+    expect(() => BrowserUtils.expectNoAttributeValue('//NotExist', 'method', 'post')).to.throw(Error);
+  });
+
+  it('Case attribute not found', () => {
```
Comment (FelixZilber) [inline src/test/specs/ExpectNoAttributeValueSpec.ts]: suggestion for name : "Not existing attribute"

File: src/test/specs/ExpectNoAttributeValueSpec.ts [hunk 3]
```diff
@@ -18,4 +18,14 @@ describeCommon('expectNoAttributeValue', () => {
   it('Exact match error thrown', () => {
     expect(() => BrowserUtils.expectNoAttributeValue(EMPTY_DIV, 'align', 'center')).to.throw(Error);
   });
+
+  it('Case Element not found.', () => {
+    BrowserUtils.navigateToUrl(sampleAppUrl);
+    expect(() => BrowserUtils.expectNoAttributeValue('//NotExist', 'method', 'post')).to.throw(Error);
```
Comment (FelixZilber) [inline src/test/specs/ExpectNoAttributeValueSpec.ts]: add error message validation

File: src/test/specs/ExpectNoAttributeValueSpec.ts [hunk 4]
```diff
@@ -18,4 +18,14 @@ describeCommon('expectNoAttributeValue', () => {
...
+  });
+
+  it('Case attribute not found', () => {
+    BrowserUtils.navigateToUrl(sampleAppUrl);
+    expect(() => BrowserUtils.expectNoAttributeValue('//form', 'NotExist', 'post')).to.throw(Error);
```
Comment (FelixZilber) [inline src/test/specs/ExpectNoAttributeValueSpec.ts]: add error message validation

File: src/test/specs/ExpectNoAttributeValueSpec.ts [hunk 5]
```diff
@@ -18,4 +18,14 @@ describeCommon('expectNoAttributeValue', () => {
   it('Exact match error thrown', () => {
     expect(() => BrowserUtils.expectNoAttributeValue(EMPTY_DIV, 'align', 'center')).to.throw(Error);
```
Comment (FelixZilber) [inline src/test/specs/ExpectNoAttributeValueSpec.ts]: add error message validation

File: src/test/specs/ExpectNoAttributeValueSpec.ts (hunk 1 above)
Comment (Ad1B3nAr0ya) [reply src/test/specs/ExpectNoAttributeValueSpec.ts]: changed to "Element not exists"

File: src/test/specs/ExpectNoAttributeValueSpec.ts (hunk 2 above)
Comment (Ad1B3nAr0ya) [reply src/test/specs/ExpectNoAttributeValueSpec.ts]: changed to : "Attribute not exists"

File: src/test/specs/ExpectNoAttributeValueSpec.ts (hunk 3 above)
Comment (Ad1B3nAr0ya) [reply src/test/specs/ExpectNoAttributeValueSpec.ts]: done

File: src/test/specs/ExpectNoAttributeValueSpec.ts (hunk 4 above)
Comment (Ad1B3nAr0ya) [reply src/test/specs/ExpectNoAttributeValueSpec.ts]: done

File: src/test/specs/ExpectNoAttributeValueSpec.ts (hunk 5 above)
Comment (Ad1B3nAr0ya) [reply src/test/specs/ExpectNoAttributeValueSpec.ts]: Unable to catch error message for validation.

Comment (FelixZilber) [reply src/test/specs/ExpectNoAttributeValueSpec.ts]: because it has `).to.throw(Error).to.throw(Error);` please fix

Comment (Ad1B3nAr0ya) [reply src/test/specs/ExpectNoAttributeValueSpec.ts]: fixed

File: src/test/specs/ExpectNoAttributeValueSpec.ts [hunk 6]
```diff
@@ -1,27 +1,34 @@
...
+
+  it('Element not exists', () => {
+    BrowserUtils.navigateToUrl(sampleAppUrl);
+    expect(() => BrowserUtils.expectNoAttributeValue('//NotExist', 'method', 'post')).to.throw(Error).with.property('message')
+      .contains("Element not exist");
+  });
+
+  it('Attribute not exists', () => {
+    BrowserUtils.navigateToUrl(sampleAppUrl);
+    expect(() => BrowserUtils.expectNoAttributeValue('//form', 'NotExist', 'post')).to.throw(Error).with.property('message')
```
Comment (FelixZilber) [inline src/test/specs/ExpectNoAttributeValueSpec.ts]: this test should check no existing attribute on existing element with different error message

Comment (Ad1B3nAr0ya) [reply src/test/specs/ExpectNoAttributeValueSpec.ts]: done

**Pull request merged**